processor/cumulativetodeltaprocessor/config.go

```diff
@@ -21,14 +21,21 @@ import (
 	"go.opentelemetry.io/collector/component"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/filter/filterset"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/cumulativetodeltaprocessor/internal/tracking"
 )
 
 // Config defines the configuration for the processor.
 type Config struct {
-
 	// MaxStaleness is the total time a state entry will live past the time it was last seen. Set to 0 to retain state indefinitely.
 	MaxStaleness time.Duration `mapstructure:"max_staleness"`
 
+	// InitialValue determines how to handle the first datapoint for a given metric. Valid values:
+	//
+	//   - auto: (default) send the first point iff the startime is set AND the starttime happens after the component started AND the starttime is different from the timestamp
```

Comment:
I believe we need to update https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/6eb5eff7ce198c56b82903cab7ee43729ba5af64/processor/cumulativetodeltaprocessor/factory.go#L39 in order for `auto` to be used as default.

Reply:
I don't think so, auto was defined as the first value `= iota`, with the real value of `0`, so it will be default if we don't set anything